### DISCUSSION TITLE
2.3.9 goals

### DISCUSSION THREAD

**Discussion by dankamongmen** [dankamongmen/notcurses#1851] [General]
# 2.3.9, mid-july 2021

next: [2.3.11](https://github.com/dankamongmen/notcurses/discussions/1905)

The core goals of 2.3.9 are to get CLI mode (see #1853) working for the standard plane (`notcurses-info` ought work everywhere), and to rigourize the input layer, so that we can reliably process control sequences while in rendered mode.

## 2.3.9 objectives
* #1837
  * and subbug #1880
* #1439 
* #1649 
* #1865
* #1692

## shit that's come up
* #1875
* #1872
* #1878
* #1894

**Comment by joseluis** [comment]
I really like this next version summary/overview posts format you've come up with

**Comment by dankamongmen** [comment]
let's get some testing on this, and it's ready to go
